### PR TITLE
allow specifying kubeconfig

### PR DESCRIPTION
**Pull request by mwei0210**

### CODE REVIEW

File: knsk.sh [hunk 1]
```diff
@@ -97,6 +98,13 @@
         TIME=$1
         shift
       ;;
+      --kubeconfig)
+        shift
+        # Check if the port is a number
```
Comment (thyarles) [inline knsk.sh:103]: Hey @mwei0210,

Awesome, it is really a usefull option! Could you just, please, fix this comment too?

Thank your for helping make this script better. Even with the news versions of k8s, this script still be useful.

Comment (mwei0210) [reply knsk.sh]: Fixed the comment and tested working. Thanks for the nice script too!

**Comment by mwei0210** [comment]
gonna close this and open a new one, forced updated changes not reflected  :(

**Comment by mwei0210** [comment]
#17

**Pull request closed without merging**